devel/pulp/devel/test_runner.py

```diff
@@ -60,9 +69,28 @@ def run_tests(packages, tests_all_platforms, tests_non_rhel5):
                      ','.join(packages)])
 
     # don't run the server or plugins tests in RHEL5.
+    flake8_exit_code = 0
     if sys.version_info >= (2, 6):
+        # make sure we test everything
         args.extend(tests_non_rhel5)
+
+        # Check the files for coding conventions
+        if run_flake8:
+            if not flake8_paths:
+                flake8_paths = ['.']
```

Comment:
We should probably find a better way of choosing the path. This assumes that your CWD is a particular value, but the run-tests.py script does not otherwise assume that. You may need to have this method require these paths to be set to avoid this problem since it will be difficult to determine otherwise.